term-size.c

```diff
@@ -1,10 +1,9 @@
 #include <stdio.h>
-#include <string.h> // strerror
-#include <errno.h>  // errno
-#include <fcntl.h>  // open(), O_RDONLY
-#include <unistd.h> // close()
-#include <curses.h> // setupterm(), tigetnum()
-#include <term.h>   // ^ Same as above
+#include <string.h>    // strerror
+#include <errno.h>     // errno
+#include <fcntl.h>     // open(), O_RDONLY
+#include <unistd.h>    // close()
+#include <sys/ioctl.h> // ioctl()
 
 int main() {
 	int tty_fd = open("/dev/tty", O_RDONLY);
```

Comment:
Pragmatically, no. Technically, yes. In fact, using `O_EVENTONLY` would probably help too to indicate we don't need any data from the device. I'm sure a someone with more knowledge on the implementation of TTY devices is going to know more about it but signaling we need the least amount of data (exactly none) doesn't hurt.